stan/math/rev/fun/mdivide_left_tri.hpp

```diff
@@ -285,11 +285,11 @@ class mdivide_left_tri_vd_vari : public vari {
     } else {
 #endif
       adjA.noalias()
-          = -Map<Matrix<double, R1, C1> >(A_, M_, M_)
+          = -Map<Matrix<double, R1, C1>>(A_, M_, M_)
                  .template triangularView<TriView>()
                  .transpose()
                  .solve(adjC
-                        * Map<Matrix<double, R1, C2> >(C_, M_, N_).transpose());
+                        * Map<Matrix<double, R1, C2>>(C_, M_, N_).transpose());
 #ifdef STAN_OPENCL
```

Comment:
I may have broken OpenCL stuff here. Not sure. I want to delete these varis though.